examples/lima/coder.yaml

```diff
@@ -81,6 +87,8 @@ provision:
       usermod -aG docker coder
       # Ensure coder listens on all interfaces
       sed -i 's/CODER_ADDRESS=.*/CODER_ADDRESS=0.0.0.0:3000/' /etc/coder.d/coder.env
+      # Also set the access URL to host.lima.internal for fast deployments
+      sed -i 's#CODER_ACCESS_URL=.*#CODER_ACCESS_URL=http://host.lima.internal:3000#' /etc/coder.d/coder.env
```

Comment:
Does this make the other change unnecessary? The one where we do:

```
sed -i 's/host.lima.internal.*/host.lima.internal host.docker.internal/' /etc/hosts
```

Reply:
> Does this make the other change unnecessary? The one where we do:
> 
> ```
> sed -i 's/host.lima.internal.*/host.lima.internal host.docker.internal/' /etc/hosts
> ```

I'm not sure, but I'd prefer to keep the alias for `host.docker.internal` around regardless.